src/Common/Form/TitleType.php

```diff
@@ -21,4 +21,9 @@ public function configureOptions(OptionsResolver $resolver): void
             ]
         );
     }
+
+    public function getBlockPrefix(): string
```

Comment:
This shouldn't be necessary here, `TitleType` will use `title` as a block prefix by default

Reply:
it won't because the parent has it hardcoded as text